vault/core.go

```diff
@@ -3331,3 +3349,22 @@ func (c *Core) CheckPluginPerms(pluginName string) (err error) {
 	}
 	return err
 }
+
+// ListMounts will provide a slice containing a deep copy each mount entry
+func (c *Core) ListMounts() ([]*MountEntry, error) {
+	c.mountsLock.RLock()
+	defer c.mountsLock.RUnlock()
+
+	var entries []*MountEntry
+
+	for _, entry := range c.mounts.Entries {
+		clone, err := entry.Clone()
+		if err != nil {
+			return nil, err
+		}
+
+		entries = append(entries, clone)
+	}
+
+	return entries, nil
+}
```

Comment:
This doesn't look related to activity log stuff. Is this correct?

Reply:
I had to pull this in (see: https://github.com/hashicorp/vault/pull/21210/commits/4dde07d5c6be66e58b66d9c307230b00066ed2c3) for one of the activitylog tests.